src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -359,10 +362,6 @@ int main(int argc, char * argv[])
         }
     }
 
-    gpCommandSender->Shutdown();
-    chip::app::InteractionModelEngine::GetInstance()->Shutdown();
-    ShutdownChip();
```

Comment:
This might be the problem of previous unsuitability issue, if the CHIP stack is shutdown before the CRMP complete the pending re-transmission, then we could see the crash

Reply:
That is the case, we should determine if there are any cases that we need to make these global variables able to be released cleanly.